models/repo.go

```diff
@@ -1004,6 +1004,13 @@ type CreateRepoOptions struct {
 	MirrorInterval string
 }
 
+// ForkRepoOptions contains the fork repository options
+type ForkRepoOptions struct {
+	OldRepo     *Repository
```

Comment:
The name `OldRepo` is not right.

Reply:
Sure, I had considered `OldRepo` or `ForkRepo` and stuck with the former to match the argument name, but IMO `OriginalRepo` (or maybe even `BaseRepo`?) is much more descriptive.

Reply:
I like personlly `OriginalRepo` or `SourceRepo`.

Reply:
`models.Repo` has `BaseRepo` and `BaseRepoID` which is the id/repo of that was forked from.

I think therefore this should be `BaseRepo`.

Reply:
I've switchedit to `BaseRepo` for consistency with the model.